frontend/src/composables/useRpcUrls.ts

```diff
@@ -0,0 +1,8 @@
+import { ChainConfigMapping } from '@/types/config';
+
+export function useRpcUrls(chains: ChainConfigMapping) {
+  const rpcUrlsList = Object.keys(chains).reduce((prev, curr) => {
```

Comment:
Also the naming "list" seems not appropriate to me, because we are returning an object.